src/decrypter.js

```diff
@@ -52,114 +50,138 @@ ntoh = function(word) {
 };
 
 /**
- * Schedule out an AES key for both encryption and decryption. This
- * is a low-level class. Use a cipher mode to do bulk encryption.
+ * Expand the S-box tables.
  *
- * @constructor
- * @param key {Array} The key as an array of 4, 6 or 8 words.
+ * @private
  */
-AES = function (key) {
-  this._precompute();
+const precompute = function() {
+  let _tables = [[[], [], [], [], []], [[], [], [], [], []]];
```

Comment:
if it's just a local variable and doesn't get exported, it doesn't need any prefix or postfix indicating that it is private. Variables are inherently private by default.